### ISSUE TITLE
playbook fails on ubuntu 14.04

### ISSUE THREAD

**Issue by levinotik** [malk/ansible-java8-oracle#2]
thanks for this helpful playbook. I tried running it on Ubuntu 14.04 LTS and got the following errors:

```
failed: [10.0.2.43] => {"cmd": "apt-key adv --recv-keys --keyserver hkp://keyserver.ubuntu.com:80 7B2C3B0889BF5709A105D03AC2518248EEA14886", "failed": true, "rc": 2}
stderr: gpg: requesting key EEA14886 from hkp server keyserver.ubuntu.com
gpg: no writable keyring found: eof
gpg: error reading `[stream]': general error
gpg: Total number processed: 0

stdout: Executing: gpg --ignore-time-conflict --no-options --no-default-keyring --homedir /tmp/tmp.5jwpbSRNqt --no-auto-check-trustdb --trust-model always --keyring /etc/apt/trusted.gpg --primary-keyring /etc/apt/trusted.gpg --recv-keys --keyserver hkp://keyserver.ubuntu.com:80 7B2C3B0889BF5709A105D03AC2518248EEA14886

msg: gpg: requesting key EEA14886 from hkp server keyserver.ubuntu.com
gpg: no writable keyring found: eof
gpg: error reading `[stream]': general error
gpg: Total number processed: 0

FATAL: all hosts have already failed -- aborting
```

**Comment by malk** [comment]
oh O.o has been a long while since I've used ubuntu at all, I shall kick a virtual machine tonight (Paris time) to check that out

**Comment by eddiejaoude** [comment]
Just to let you know, I added this role to my playbook for ubuntu and it worked fine. Thanks for the share! :+1:

**Comment by malk** [comment]
yeah I could not reproduce it either O.o

**Comment by malk** [comment]
do you have a vagrant of your problem @levinotik ?

**Comment by palexander** [comment]
The command may require `sudo` on some systems (as it did with mine). The error I got was exactly what @levinotik saw from a box with a non-root user. Adding `sudo: yes` to the ansible command for adding the Oracle Java apt repo fixed it.

**Comment by malk** [comment]
care to send a PR of your working version @palexander ?

**Comment by malk** [comment]
solved by #4
